packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -261,6 +295,14 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
                 this.contexts.set(path, channelContext);
                 this.contextsDeferred.set(path, deferred);
             });
+
+            if (initializeEntrypoint) {
```

Comment:
i was thinking we would internally substitute request here if initializeEntrypoint is underfined, so all paths have the same behavior, rather than load being different than new

Reply:
Ah, that seems reasonable, yeah. Let me give it a shot... after I fix the location of this bit, I mistakenly put it inside an `if`, not directly in the constructor. That could be related to a bunch of failing tests in CI.

Reply:
Actually, not moving forward with the idea to use request as a fallback for `initializeEntryPoint` per https://github.com/microsoft/FluidFramework/pull/11887#discussion_r973242692.